src/main/java/humbledude/carracing/Car.java

```diff
@@ -2,15 +2,35 @@
 
 public class Car {
 
+    private static final String DEFAULT_CAR_NAME = "car";
+
+    private String name = DEFAULT_CAR_NAME;
     private int position = 0;
 
+    public Car() {}
```

Comment:
디폴트 생성자를 추가해주신 이유가 있을까요 :) ?
해당 미션의 자동차는 이름을 가져야 비로소 인스턴스가 완성된다고 생각하기에 name을 받는 생성자만 사용하거나 디폴트 생성자를 통해 주 생성자를 호출하는 식으로 변경하면 좋을 것 같은데 어떻게 생각하시나요 :) ?

```suggestion
    private final String name;
    private int position = 0;

    public Car() {
        this(DEFAULT_CAR_NAME);
    }
```

Reply:
기존 테스트 코드에서 자동차에 이름을 정하지 않고 사용하던 부분이 있었어요.
거기를 다 이름을 지어주는 코드로 바꿀까 하다가~
요구사항에는 없었지만, 딱히 default 이름을 갖는것도 나쁘지 않다 라고 생각을 해서 default 이름을 갖도록 생성자를 만들었어요

디폴트 생성자에서 name 파라미터를 갖는 생성자를 호출한다는 방법은 일단 생각도 안해봤구요 ㅎㅎㅎㅎㅎ
final 키워드를 통해 이값은 불변이다 라고 표현되는 부분이 괜찮은거 같네요~